app/api/routers/breeze_buddy/numbers/handlers.py

```
@@ -0,0 +1,202 @@
+        )
+
+        if outbound_number:
+            logger.info(
+                f"Outbound number {number.number} created successfully with ID {outbound_number.id}"
+            )
+            return outbound_number
+        else:
+            raise HTTPException(
+                status_code=status.HTTP_400_BAD_REQUEST,
+                detail="Failed to create outbound number"
+            )
```

Ruff (0.14.8)

Abstract `raise` to an inner function

(TRY301)